docs/tracking.md

```diff
@@ -51,22 +83,55 @@ The parser supports reading a folder containing multiple trainings with a struct
             └─ …
 ```
 
+## Weight & Biases dashboard
 
-The following rules are applied:
-* `./models` sub-folders are projects (e.g. `en-sv`), corresponding to projects in W&B.
-* Projects contains multiple groups (e.g. `opusmt-multimodel-test`), each containing multiple runs (e.g. `student-finetuned`) and usually an `evaluation` folder.
-* For each run, `train.log` is parsed (`valid.log` results are usually contained in `train.log`) and published to W&B.
-* `.metrics` files in the `evaluation` are parsed (looking for one float value per line) and also published on the same run (e.g. `[metric] tc_Tatoeba-Challenge-v2021-08-07`).
-* Once all runs of a group have been published, a last group is pushed to W&B, named `group_logs`. That run contains no metrics but all experiment files published as artifacts.
+The publication is handled via the extensible module `translations_parser.publishers`.
 
-### Publish from Taskcluster
+### Structure
 
-The parser supports reading training tasks directly from the Taskcluster API (no authentication).
-The results are published the same way as for experiments folder.
+Runs on Weight & Biases are groupped by expermient. The group is suffixed by the complete Taskcluster group ID, and each of its runs (train or evaluation) is prefixed by the first 5 characters. This is required to compare runs with similar name (e.g. `teacher-1`) among different groups.
 
-You can parse a group (with other traversal tasks) by running:
+### Training data
+
+Metrics parsed in real time during the training are published in the **Charts** section of Weight & Biases.
+
+![Training charts](img/tracking/training_charts.png)
+
+Training runs have their Marian and Opustrainer configuration published to the **Overview** section in Weight & Biases.
```

Comment:
It's worth clarifying here the categories we came up with (model, arguments, marian, opustrainer, training), what they mean and where those configs come from.